plugin-server/src/kafka/consumer.ts

```diff
@@ -138,11 +152,16 @@ export class KafkaConsumer {
         this.config.autoCommit ??= true
         this.config.autoOffsetStore ??= true
         this.config.callEachBatchWhenEmpty ??= false
+        this.config.waitForBackgroundTasksOnRebalance ??= false
```

Comment:
Could we make this read from defaultConfig instead of the given config? It's more of a general config option liek the fetchBatchSize for example so feels better there as we can then apply this to any service (would love to test the CDP stuff too for example)